### DISCUSSION TITLE
See a project you love? Make a copy and get started faster than ever!

### DISCUSSION THREAD

**Discussion by 2percentsilk** [community/community#44815] [Projects and Issues]
### Select Topic Area

Show & Tell

### Body

We've shipped the ability to copy your project's views, custom fields, and draft issues. Check out [our docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/copying-an-existing-project) for more and leave us any feedback you've got!

**Comment by jeffwilcox** [comment]
Hi - I think the "clone project" GraphQL mutation is for V1 projects, but I would be wrong... is this new project copy feature exposed via the mutation https://docs.github.com/en/graphql/reference/mutations#cloneproject?

**Reply by mntlty** [reply]
@jeffwilcox that's correct, GraphQL operations for ProjectsV2 have that in the name. We have a copyProjectV2 mutation that will be available soon!

**Comment by axecopfire** [comment]
We're currently iterating through a list of issues in a template repository and generating project items that way. This is a nice feature

**Comment by jarmak-nv** [comment]
Hi! Really appreciate this functionality, it's a big help for us since it let's people get started quickly and we can create a template for things like long-term projects, or short-term and have them pre-configured as we see fit. A couple thoughts:


- Insights don't appear to copy
   - This would be useful because we may want to set specific insight/filters and that setup can be a bit "in the weeds" for someone who is expecting to just copy a template and get it running
- Same for workflows
   - Things like auto set to `To do` are nice-to-haves, again not a blocker because we can just note to people in the template README what they need to do, but I'd like for that section to be empty
   - The auto-add-issue workflow poses a challenge here since projects are org-level and not repo-level
      - Long term nice to have would be a guided process saying something like "the template has an auto-add workflow set up, would you like to connect a repo for auto-adding?"

**Reply by rileybroughten** [reply]
👋 Thanks @jarmak-nv for this feedback! I'll take this back to the team and will follow up if we have any questions 👍

**Comment by yanghua-ola** [comment]
This is great! I have a set of http scripts to do pretty much the same thing. Speaking of which, it would be wonderful if there is an option to export/create project with a manifest. That gives great freedom for users to share/tweak project templates.
